src/Lti/Service/UserGenerator/UserGeneratorDatabaseStorage.php

```diff
@@ -24,28 +24,43 @@
 
 use Doctrine\ORM\OptimisticLockException;
 use Doctrine\ORM\ORMException;
+use OAT\SimpleRoster\DataTransferObject\AssignmentDto;
+use OAT\SimpleRoster\DataTransferObject\AssignmentDtoCollection;
+use OAT\SimpleRoster\DataTransferObject\UserDto;
+use OAT\SimpleRoster\DataTransferObject\UserDtoCollection;
+use OAT\SimpleRoster\Entity\User;
 use OAT\SimpleRoster\Lti\Model\AutogeneratedUser;
 use OAT\SimpleRoster\Model\AssignmentCollection;
 use OAT\SimpleRoster\Model\UserCollection;
-use OAT\SimpleRoster\Repository\AssignmentRepository;
 use OAT\SimpleRoster\Repository\Criteria\FindUserCriteria;
+use OAT\SimpleRoster\Repository\NativeAssignmentRepository;
+use OAT\SimpleRoster\Repository\NativeUserRepository;
 use OAT\SimpleRoster\Repository\UserRepository;
 use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
 
 class UserGeneratorDatabaseStorage implements UserGeneratorStateStorageInterface
```

Comment:
The number of objects is breaking the Jenkins pipeline, don't think there's a fix for that other than using less or increasing the limit (which, I believe, won't be the best idea)